### PR TITLE
[IMPAC-408] Access pusher_key from IMPAC_CONFIG

### PR DESCRIPTION
**Pull request by xaun**
@alexnoox please review

### CODE REVIEW

**Comment by xaun** [comment]
https://github.com/maestrano/mno-enterprise/pull/163

**Comment by xaun** [comment]
https://github.com/maestrano/impac-express/pull/12

File: src/app/impac.config.coffee [hunk 1]
```diff
@@ -56,10 +56,14 @@ angular.module 'mnoEnterpriseAngular'
...
 #======================================================================================
-.run((ImpacLinking, ImpacConfigSvc) ->
+.run((ImpacLinking, ImpacConfigSvc, IMPAC_CONFIG) ->
   data =
     user: ImpacConfigSvc.getUserData
     organizations: ImpacConfigSvc.getOrganizations
 
+  pusherKey = IMPAC_CONFIG.pusher.key if IMPAC_CONFIG.pusher && IMPAC_CONFIG.pusher.key
+
+  angular.merge(data, {pusher_key: pusherKey}) if pusherKey
```
Comment (alexnoox) [inline src/app/impac.config.coffee]: Can't those 2 lines be written like this:
```coffeescript
data.pusher_key = IMPAC_CONFIG.pusher.key if IMPAC_CONFIG.pusher? && IMPAC_CONFIG.pusher.key?
```

**Comment by xaun** [comment]
@alexnoox yep absolutely right on the syntax there. I have pushed a change.

**Comment by alexnoox** [comment]
@xaun @cesar-tonnoir Merged

**Pull request merged**